CLAUDE.md

```
@@ -0,0 +1,71 @@
+
+### Versioning & Publishing
+```bash
+pnpm changeset            # Create a changeset for version bump
+pnpm version              # Apply changesets, update CHANGELOG
+pnpm release              # Publish all packages to npm
+```
+
+## Architecture
+
+### Code Generation Flow
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)